source/extensions/retry/priority/previous_priorities/previous_priorities.cc

```diff
@@ -51,7 +51,7 @@ bool PreviousPrioritiesRetryPriority::adjustForAttemptedPriorities(
   // This allows us to fall back to the unmodified priority load when we run out of priorities
   // instead of failing to route requests.
   if (total_availability == 0) {
-    for (auto&& excluded_priority : excluded_priorities_) {
+    for (auto excluded_priority : excluded_priorities_) {
```

Comment:
is this for loop working as intended? 
I think it's not reseting the status of the excluded_priorities_ before/after the change?